Library/Formula/json-fortran.rb

```diff
@@ -15,9 +15,9 @@ class JsonFortran < Formula
 
   option "with-unicode-support", "Build json-fortran to support unicode text in json objects and files"
   option "without-test", "Skip running build-time tests (not recommended)"
-  option "without-robodoc", "Do not build and install ROBODoc generated documentation for json-fortran"
+  option "without-ford", "Do not build and install FORD generated documentation for json-fortran"
```

Comment:
@dunn, I can’t seem to determine the answer to this question from  the Formula cookbook page:

> How do I treat the ford dependency if I change `without-ford` to `without-docs`?

Would my `depends_on "ford"` line look like this:
```ruby
depends_on "ford" => ["with-docs", :recommended, :build]
```
?